.github/workflows/unmarked_tests.yaml

```diff
@@ -0,0 +1,46 @@
+name: Unmarked Tests Checker
+
+on:
+  workflow_call:
+    inputs:
+      project_root:
+        required: true
+        type: string
+
+permissions:
+  contents: read
+
+jobs:
+  pytest-linux:
+      runs-on: ubuntu-latest
+
+      steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v3
+        with:
+          python-version: "3.10"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install pytest-custom-exit-code
+          pip install .
+      - name: Capture pytest output
+        id: collect_tests
+        run: |
+          pytest_output=$(pytest --suppress-no-test-exit-code -vv -m "not unit and not integration and not end2end" --collect-only)
```

Comment:
I would add in the readme something about expecting the markers to be called 'unit' 'integration' and 'end2end' because that is technically decided in their repo, not this one